File: PYTHON/boca/resolvidos cpp/mauricio/estados.cpp
#include <iostream>
#include <string>
using namespace std;

int main(){
	string estado;
	cin >> estado;
	if(estado=="roraima"||estado=="acre"||estado=="amapa"||estado=="amazonas"||estado=="para"||estado=="rondonia"||estado=="tocantins"){
		cout << "Regiao Norte\n";
	}
	else{
		cout << "Outra Regiao\n";
	}
	return 0;
}

File: PYTHON/boca/resolvidos cpp/talita/estados.py
estado = input()
regiao_norte = ["roraima", "acre", "amapa", "amazonas", "para","rondonia", "tocantins"]

contador = 0

for est in regiao_norte: 
    if est == estado: 
        print("Regiao Norte") 
    else: 
        contador = contador + 1 

if contador == 7: 
    print("Outra regiao")



"""
python tem break!!!         break  # Para o loop!
"""

"""-----------------------versao COPILOT--------------------------------------"""
estado = input()
regiao_norte = ["roraima", "acre", "amapa", "amazonas", "para", "rondonia", "tocantins"]

if estado in regiao_norte:
    print("Regiao Norte")
else:
    print("Outra regiao")
